packages/integration-sdk-http-client/src/util.ts

```diff
@@ -18,3 +18,44 @@ export function isNonRetryableError(statusCode: number): boolean {
   }
   return false;
 }
+
+export function isRetryableStatusCode(statusCode: number): boolean {
+  if (
+    statusCode === 429 ||
+    statusCode === 408 ||
+    (statusCode >= 500 && statusCode !== 501)
+  ) {
+    return true;
+  }
+  return false;
+}
```

Comment:
I like framing logic in a positive sense, but I know we have thought about just retrying everything. @VDubber I would love to talk to you more about how we can do this better.

Reply:
I'd be interested in discussing this as well.  I think as long as we can treat this as a default and let users of the client override it, we're fine.

What you have currently for the retryable/non-retryable codes seems reasonable to me.